trace/HTTPAttributes.md

```diff
@@ -0,0 +1,35 @@
+# HTTP Attributes
+
+This document summarizes HTTP attributes recorded from HTTP
+requests and responses.
+
+HTTP integrations should set the following attributes on the client
+and server spans. For a server, request represents the incoming request.
+For a client, request represents the outgoing request.
+
+```
+| Attribute name       | Description            | Example value                   |
+|----------------------|------------------------|---------------------------------|
+| "http.url"           | Request URL            | "https://example.com/users?q=a" |
+| "http.host"          | Request URL host       | "example.com"                   |
+| "http.method"        | Request URL method     | "GET"                           |
+| "http.path"          | Request URL path       | "/users"                        |
```

Comment:
host and path are already available from the url, what's the rationale for capturing them again? Tracing backends can easily extract them later.

Reply:
Or have only host and path and leave out url.

Reply:
Due to the security concerns, we should prefer having host and path instead of requiring URL to be recorded. I removed the URL attribute and also marked all attributes as optional. PTAL.